ibis/expr/types/arrays.py

```diff
@@ -386,6 +387,22 @@ def map(self, func: Callable[[ir.Value], ir.Value]) -> ir.ArrayValue:
         │ [4]                  │
         │ []                   │
         └──────────────────────┘
+
```

Comment:
I am prioritizing Deferreds in the examples because I think they are gonna be the most common usage. But could swap the order if you want the lambdas to get more attention.

Reply:
Keeping deferreds first is fine!